packages/bpk-component-calendar/src/utils.js

```diff
@@ -21,21 +21,33 @@ import {
   calendarDaySpacing,
 } from 'bpk-tokens/tokens/base.es6';
 
-const remToPx = value => {
-  let parsed = null;
+const splitToken = value => {
+  const match = value.match(/(^[+-]?(?:\d*\.)?\d+)(.+)/i);
```

Comment:
This regex should be pulled out of this function to avoid it being recreated at all times

Reply:
good point, I'll do

Reply:
done